tests/src/Unit/CollaboraDiscoveryTest.php

```diff
@@ -28,12 +29,167 @@ public function testWopiClientUrl(): void {
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        'application/vnd.sun.xml.draw',
+        'application/x-dbase',
+        'application/x-dif-document',
+        'text/csv',
+        'text/plain',
+        'text/rtf',
+        'text/spreadsheet',
+        'writer-web',
+      ],
+      'edit,view' => [
+        'calc',
+        'impress',
+        'writer',
+        'writer-global',
+      ],
+      'edit,view,view_comment' => [
+        'draw',
+      ],
+      'view' => [
+        'application/clarisworks',
+        'application/coreldraw',
+        'application/macwriteii',
+        'application/vnd.lotus-1-2-3',
+        'application/vnd.ms-excel.template.macroEnabled.12',
+        'application/vnd.ms-powerpoint.template.macroEnabled.12',
+        'application/vnd.ms-visio.drawing',
+        'application/vnd.ms-word.template.macroEnabled.12',
+        'application/vnd.ms-works',
+        'application/vnd.oasis.opendocument.graphics-template',
+        'application/vnd.oasis.opendocument.presentation-template',
+        'application/vnd.oasis.opendocument.spreadsheet-template',
+        'application/vnd.oasis.opendocument.text-master-template',
+        'application/vnd.oasis.opendocument.text-template',
+        'application/vnd.openxmlformats-officedocument.presentationml.template',
+        'application/vnd.openxmlformats-officedocument.spreadsheetml.template',
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.template',
+        'application/vnd.sun.xml.calc',
+        'application/vnd.sun.xml.calc.template',
+        'application/vnd.sun.xml.chart',
+        'application/vnd.sun.xml.draw.template',
+        'application/vnd.sun.xml.impress',
+        'application/vnd.sun.xml.impress.template',
+        'application/vnd.sun.xml.writer',
+        'application/vnd.sun.xml.writer.global',
+        'application/vnd.sun.xml.writer.template',
+        'application/vnd.visio',
+        'application/vnd.visio2013',
+        'application/vnd.wordperfect',
+        'application/x-abiword',
+        'application/x-aportisdoc',
+        'application/x-fictionbook+xml',
+        'application/x-gnumeric',
+        'application/x-hwp',
+        'application/x-iwork-keynote-sffkey',
+        'application/x-iwork-numbers-sffnumbers',
+        'application/x-iwork-pages-sffpages',
+        'application/x-mspublisher',
+        'application/x-mswrite',
+        'application/x-pagemaker',
+        'application/x-sony-bbeb',
+        'application/x-t602',
+        'image/bmp',
+        'image/cgm',
+        'image/gif',
+        'image/jpeg',
+        'image/jpg',
+        'image/png',
+        'image/svg+xml',
+        'image/tiff',
+        'image/vnd.dxf',
+        'image/x-emf',
+        'image/x-freehand',
+        'image/x-wmf',
+        'image/x-wpg',
+      ],
+      'view_comment' => [
+        'application/pdf',
+      ],
+    ], $supported_action_types);
```

Comment:
@brummbar We can remove this part of the test before we merge.
I want to keep it for now to have an overview what we will get with the existing discovery.xml from Collabora.